chicago/models.py

```diff
@@ -172,6 +172,30 @@ def most_recent_past_city_council_meeting(cls):
         else:
             return None
 
+    @property
+    def clean_agenda_items(self):
+        agenda_items = (
+            self.agenda.order_by("order").all().prefetch_related("related_entities")
+        )
+        agenda_deduped = []
+        descriptions_seen = []
+        for a in agenda_items:
+            if a.description not in descriptions_seen:
+                descriptions_seen.append(a.description)
+                agenda_deduped.append(a)
+
+        return agenda_deduped
+
+    @property
+    def video_vimeo_id(self):
+        try:
+            link = self.media.first().links.first().url
+            vimeo_id = re.match(".*?([0-9]+)$", link).group(1)
```

Comment:
assuming the links are all vimeo and that the ID is always the last part of it, which seems to be the case